frontend/src/components/ride/RideProfile.tsx

```diff
@@ -0,0 +1,112 @@
+// TODO: recreate the userform and modify for ride.
+// Make sure to look into the context and see if it is needed.
+
+import React, { useEffect, useState } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
```

Comment:
let's remove `useLocation` since it is not being used.

Reply:
Done